shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/job/FinishedCheckJob.java

```diff
@@ -51,7 +51,7 @@ public void execute(final ShardingContext shardingContext) {
                 continue;
             }
             String jobId = jobInfo.getJobId();
-            if (checkJobStatus(jobId)) {
+            if (checkAllowDataCheckJobStatus(jobId)) {
                 continue;
             }
```

Comment:
Looks `checkAllowDataCheckJobStatus` means not allow, since there's `continue;`.

Use `isXxx` style could be more certain.